### PR TITLE
add dropdown for parent ids

### PR DESCRIPTION
**Pull request by sjsalonen**
- Select parent id from dropdown such as with the types
- Show restart job button even after successful run

This closes issues #5 and #8

### CODE REVIEW

File: source/web/components/JobEditor.js [hunk 1]
```diff
@@ -268,23 +268,34 @@ export default class JobEditor extends Component {
...
                 <FormGroup label="Parents" labelFor="parents">
-                    <ControlGroup>
-                        <InputGroup
+                    <div className={`${Classes.SELECT} select-editable ${Classes.FILL}`}>
+                        <select
+                            onChange={evt => this.setState({ newParent: evt.target.value })}
+                            value={this.state.newParent}
+                        >
+                            <For each="jobId" index="idx" of={this.props.jobIds}>
+                                <option key={idx} value={jobId}>
```
Comment (perry-mitchell) [inline source/web/components/JobEditor.js]: Key could be `jobId`, so that the order isn't important. `idx` can change too easily :)

**Review by perry-mitchell — CHANGES_REQUESTED** [review]
Just one item!

File: source/web/components/JobEditor.js (hunk 1 above)
Comment (sjsalonen) [reply source/web/components/JobEditor.js]: Fixed!

**Review by perry-mitchell — APPROVED** [review]
Nice!

**Pull request merged**